pages/index/_learn-more.html

```diff
@@ -38,3 +38,16 @@ <h2>Get expert support and training.</h2>
     <p><a href="/training/" class="btn btn-primary" role="button" ga-on="click" ga-event-category="home-learn-more" ga-event-action="preview-training">Preview Our Training</a></p>
   </div>
 </div>
+
+<div class="row row-cis-compliance">
+  <div class="col-xs-10 col-xs-offset-1 col-sm-5 col-sm-push-7 col-sm-offset-0 col-md-6 col-md-push-6 col-diagram">
+    <img src="/assets/img/infrastructure-cube-icons/infrastructure-design-your-stack-grunty.png" />
+  </div>
+  <div class="col-xs-12 col-sm-7 col-sm-pull-5 col-md-6 col-md-pull-6 col-description">
+    <h2>Achieve compliance.</h2>
+    <p>Achieve compliance with the CIS AWS Foundations Benchmark. The Gruntwork compliance infrastructure as code
+    modules are certified compliant by the Center for Internet Security. Leverage our <a href="/guides">compliance
+    guide</a> for a complete walkthrough. 
```

Comment:
A slight rephrase to consider:

Gruntwork's modules are certified compliant by the Center for Internet Security (CIS), so you can get compliance out of the box, either by having Gruntwork deploy an end-to-end CIS-compliant Reference Architecture for you, or by assembling your own architecture using our modules and compliance guide.